Dockerfile

```diff
@@ -1,40 +1,59 @@
-FROM debian:jessie
+FROM python:2-slim
+MAINTAINER Mozilla Telemetry <telemetry-analysis-service@mozilla.org>
 
 ENV PYTHONUNBUFFERED 1
 ENV PYTHONPATH /app/
 
-RUN set -x \
-    && apt-get update \
-    && apt-get install locales -y \
-    && locale-gen en_US.UTF-8
-
 EXPOSE 8000
 
-RUN adduser --uid 1000 --disabled-password --gecos '' --no-create-home webdev
-
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
-        build-essential python python-dev python-pip libpq-dev \
-        postgresql-client gettext sqlite3 libffi-dev \
-    && apt-get autoremove -y \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-
+# add a non-privileged user for installing and running the application
+RUN mkdir /app && \
```

Comment:
Pre-creating the home directory as empty and with required permissions is to avoid `useradd` populating it with home directory skeleton files. Those files aren’t needed and best to leave out so they don’t interfere with anything at a later command.